ci-operator/config/openshift/ironic-image/openshift-ironic-image-master.yaml

```diff
@@ -0,0 +1,28 @@
+base_images:
+  base:
+    name: "4.2"
+    namespace: ocp
+    tag: base
```

Comment:
So in the downloader, this was base-os

https://github.com/openshift/release/pull/4411/files